docker/Dockerfile.bionic-glibc

```diff
@@ -37,8 +46,21 @@ RUN apt-get update && apt-get install -y \
       llvm-${LLVM_VERSION}-runtime \
       libllvm${LLVM_VERSION} \
       systemtap-sdt-dev \
-      python3 \
-      quilt
+      python \
+      quilt \
+      libssl-dev
+
+RUN apt remove --purge --auto-remove cmake
+RUN version=${CMAKE_VER} \
```

Comment:
I think we could probably short-circuit this with a check to the distro - only install cmake if xenial? Should speed up the build.

Reply:
fixed, let me know if you'd like me to make it a bit cleaner.